package.json

```diff
@@ -93,7 +93,6 @@
     "focus-trap-react": "^10.2.1",
     "fuse.js": "^7.0.0",
     "google-protobuf": "^3.21.2",
-    "imgur": "^2.4.2",
```

Comment:
this library doesn't work – its used in the edit profile page only and on prod it's broken with a 'unsupported file type' error. Switched to axios / standard http, works fine.

Reply:
I love removing packages 🙌🏼